gdmix-trainer/src/gdmix/util/constants.py

```diff
@@ -8,7 +8,7 @@
 OFFSET = "offset"
 SAMPLE_ID = "sample_id"
 SAMPLE_WEIGHT = "sample_weight"
-FEATURE_BAGS = "feature_bags"
+FEATURE_BAGS = "feature_bag"
```

Comment:
Change the constant name for consistency and bump the version?

Reply:
I think the file you reviewed is outdated, the latest code is `FEATURE_BAG = "feature_bag"`.